Makefile

```diff
@@ -114,7 +114,10 @@ setup-e2e-tests: anchore-ci venv ## Start kind cluster and set up end to end tes
 	@$(ACTIVATE_VENV) && $(CI_CMD) setup-e2e-tests "$(COMMIT_SHA)" "$(DEV_IMAGE_REPO)" "$(GIT_TAG)" "$(TEST_IMAGE_NAME)"
 
 test-e2e: anchore-ci venv ## Run end to end tests (assuming cluster is running and set up has been run)
-	@$(ACTIVATE_VENV) && $(CI_CMD) e2e-tests
+	@$(MAKE) test-cli
+
+test-cli: setup-test-infra venv ## Run cli-driven end to end tests (assuming cluster is running and set up has been run)
```

Comment:
What the purpose of adding a target that invokes another target? Why not just keep everything in the `test-e2e` target?

Reply:
Introducing the change gradually - although I think it doesn't matter, so I'll make the change as you suggest!